velox/serializers/PrestoSerializer.cpp

```diff
@@ -383,11 +361,13 @@ void read<StringView>(
   }
   readNulls(source, size, *flatResult);
 
-  int32_t dataSize = source->read<int32_t>();
-  const auto& stringBuffers = flatResult->stringBuffers();
-  BufferPtr strings = findOrAllocateStringBuffer(dataSize, stringBuffers, pool);
-  flatResult->setStringBuffers({strings});
-  auto rawStrings = strings->asMutable<uint8_t>();
+  const int32_t dataSize = source->read<int32_t>();
+  if (dataSize == 0) {
+    return;
+  }
+
+  auto* rawStrings =
+      flatResult->getRawStringBufferWithSpace(dataSize, true /*exactSize*/);
```

Comment:
Not sure our velox coding style. Google coding style suggests to annotate in this way
```
dataSize, /*exactSize=*/true);
```